packages/design-system-web/src/components/geov-entity-properties/geov-entity-properties.tsx

```diff
@@ -0,0 +1,206 @@
+import { Component, Host, h, Prop, State } from '@stencil/core';
+import { sparqlJson } from '../../lib/sparqlJson';
+
+const qrOntomeURI = (id: string) => `
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX ontome: <https://ontome.net/ontology/>
+PREFIX geov: <http://geovistory.org/resource/>
+
+SELECT ?object
+WHERE {
+  OPTIONAL{geov:${id} rdf:type ?object .}
+}
+LIMIT 1
+`;
+
+const qrTypeAndLabel = (id: string) => `
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX xml: <http://www.w3.org/XML/1998/namespace>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+PREFIX time: <http://www.w3.org/2006/time#>
+PREFIX ontome: <https://ontome.net/ontology/>
+PREFIX geov: <http://geovistory.org/resource/>
+
+SELECT ?subject ?subjectLabel ?objectLabel
+WHERE {
+  {SELECT ?subject WHERE {geov:${id} rdf:type ?subject.} LIMIT 1}.
+  ?subject rdfs:label ?objectLabel .
+  geov:${id} rdfs:label ?subjectLabel.
+}
+`;
+
+const qrOutgoingPropsWithCount = (id: string) => `
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX xml: <http://www.w3.org/XML/1998/namespace>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+PREFIX time: <http://www.w3.org/2006/time#>
+PREFIX ontome: <https://ontome.net/ontology/>
+PREFIX geov: <http://geovistory.org/resource/>
+
+SELECT ?predicate ?predicateLabel (count(distinct ?object) as ?count) 
+WHERE {
+  geov:${id} ?predicate ?object .
+  ?predicate rdfs:label ?predicateLabel
+}
+GROUP BY ?predicate ?predicateLabel
+LIMIT 100
+`;
+
+const qrIncomingPropsWithCount = (id: string) => `
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX xml: <http://www.w3.org/XML/1998/namespace>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+PREFIX time: <http://www.w3.org/2006/time#>
+PREFIX ontome: <https://ontome.net/ontology/>
+PREFIX geov: <http://geovistory.org/resource/>
+
+SELECT   ?predicate ?predicateLabel (count(distinct ?subject) as ?count) 
+WHERE {
+ ?subject ?predicate geov:${id} .
+ ?predicate rdfs:label ?predicateLabel
+}
+GROUP BY ?predicate ?predicateLabel
+LIMIT 100
+`;
+
+export interface Binding<T> {
```

Comment:
each sparql query has its on bindings. split this interface in 3, one for each query.